src/main/java/com/ft/methodearticlemapper/resources/PostContentToTransformResource.java

```diff
@@ -88,7 +90,9 @@ private void validateUuid(String uuid, EomFile eomFile) {
             throw ClientError.status(400).context(null).reason(ErrorMessage.UUID_REQUIRED).exception();
         }
         try {
-
+            if (!uuidPattern.matcher(uuid).matches()) {
+                throw new IllegalArgumentException();
```

Comment:
as discussed, there might be an easier way to do this, without regex. check out http://expertdevelopers.blogspot.ro/2013/07/uuid-simple-technique-for-uuid.html